File: exercicios/Python/1074/1074.py
n = int(input())
x=['']
for i in range(1,n+1):
    x.append(int(input()))
for i in range(1,n+1):
    if(x[i] == 0): print("NULL")

    if (x[i] % 2 ) == 0 and x[i] > 0:
        print("ODD POSITIVE")
    if (x[i] % 2 ) != 0 and x[i] > 0:
        print("EVEN POSITIVE")
    if (x[i] % 2 ) == 0 and x[i] < 0:
        print("ODD NEGATIVE")
    if (x[i] % 2 ) != 0 and x[i] < 0:
        print("EVEN NEGATIVE")

File: exercicios/R/1074/1074.r
input = file('stdin','r')

n = as.integer(readLines(input,n=1))

for (i in 1:n) {
   v = as.integer(readLines(input,n=1))

   if(v == 0){
       write("NULL",'')
   }
   if(v%%2 == 0 & v > 0){
       write("EVEN POSITIVE",'')
   }else if(v%%2 !=0 & v > 0){
       write("ODD POSITIVE",'')
   }
   if(v%%2 == 0 & v < 0){
       write("EVEN NEGATIVE",'')
   }else if (v%%2 !=0 & v < 0){
       write("ODD NEGATIVE",'')
   }
}